### PR TITLE
111856 Jakarta activation bundle

### PR DESCRIPTION
**Pull request by uberskigeek**
Create Jakarta activation 2.0 bundle

### CODE REVIEW

**Comment by uberskigeek** [comment]
fixes #11856

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

File: dev/com.ibm.websphere.jakarta.activation.2.0/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.websphere.jakarta.activation.2.0/bnd.bnd:2]: 2020

File: dev/com.ibm.websphere.appserver.features/visibility/public/jakartaMail-1.6/resources/l10n/com.ibm.websphere.appserver.jakartaMail-1.6.properties [hunk 2]
```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2012, 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jakartaMail-1.6/resources/l10n/com.ibm.websphere.appserver.jakartaMail-1.6.properties:2]: Should be 2020.

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
fixes #12201

**Comment by uberskigeek** [comment]
#run-libby-bot

**Comment by uberskigeek** [comment]
All Failures in the personal build are unrelated to these changes

**Pull request merged**